infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -504,6 +504,151 @@ all the operations for the connector:
   - `PSOXY_DROPBOX_BUSINESS_CLIENT_ID` with `App key` value.
   - `PSOXY_DROPBOX_BUSINESS_CLIENT_SECRET` with `App secret` value.
 
+EOT
+    },
+    jira-server = {
+      source_kind : "jira-server"
+      worklytics_connector_id : "jira-server-psoxy"
+      target_host : var.jira_server_url
+      source_auth_strategy : "oauth2_access_token"
+      display_name : "Jira REST API"
+      identifier_scope_id : "jira"
+      worklytics_connector_name : "Jira Server REST API via Psoxy"
+      secured_variables : [
+        { name : "ACCESS_TOKEN", writable : false },
+      ],
+      reserved_concurrent_executions : null
+      example_api_calls_user_to_impersonate : null
+      example_api_calls : [
+        "/rest/api/2/search?maxResults=25",
+        "/rest/api/2/issue/${var.example_jira_issue_id}/comment?maxResults=25",
+        "/rest/api/2/issue/${var.example_jira_issue_id}/worklog?maxResults=25",
+        "/rest/api/latest/search?maxResults=25",
+        "/rest/api/latest/issue/${var.example_jira_issue_id}/comment?maxResults=25",
+        "/rest/api/latest/issue/${var.example_jira_issue_id}/worklog?maxResults=25",
+      ],
+      external_token_todo : <<EOT
+1. Follow the instructions to create a [Personal Access Token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) in your instance
+2. Disable or mark a proper expiration of the token.
+3. Copy the value of the token in PSOXY_JIRA_SERVER_ACCESS_TOKEN variable as part of AWS System Manager parameters store / GCP Cloud Secrets (if default implementation)
+NOTE: If your token has been created with expiration date, please remember to update it before that date to ensure connector is going to work.
+EOT
+    }
+    jira-cloud = {
+      source_kind : "jira-cloud"
+      worklytics_connector_id : "jira-cloud-psoxy"
+      target_host : "api.atlassian.com"
+      source_auth_strategy : "oauth2_refresh_token"
+      display_name : "Jira REST API"
+      identifier_scope_id : "jira"
+      worklytics_connector_name : "Jira REST API via Psoxy"
+      secured_variables : [
+        { name : "ACCESS_TOKEN", writable : true },
+        { name : "REFRESH_TOKEN", writable : true },
+        { name : "CLIENT_ID", writable : false },
+        { name : "CLIENT_SECRET", writable : false },
+        { name : "OAUTH_REFRESH_TOKEN", writable : true }, # Lock for sharing access token, handled by Psoxy
```

Comment:
And Jira Cloud uses the access token + refresh + token; as refresh_token is invalidated once a new one is requested, access_token needs to be shared between instances. For that, `OAUTH_REFRESH_TOKEN` is a variable used as a lock stuff

Reply:
I'm very confused. Why isn't this the same as `REFRESH_TOKEN`? imho, `REFRESH_TOKEN` should be the canonical oauth one. If JIRA  cloud has some additional token that doesn't work like canonical oauth,  we should give it some more specific name